iocore/net/QUICNetVConnection.cc

```diff
@@ -607,6 +687,30 @@ QUICNetVConnection::state_connection_draining(int event, Event *data)
 
   QUICErrorUPtr error = QUICErrorUPtr(new QUICNoError());
   switch (event) {
+  case QUIC_EVENT_SHUTDOWN:
+    this->_unschedule_closing_timeout();
```

Comment:
These blocks are really similar to the one in state_connection_closing(). Please make a function for this.

Reply:
sounds reasonable!